requirements/lint.txt

```diff
@@ -8,10 +8,6 @@ appdirs==1.4.4
     # via
     #   black
     #   virtualenv
-attrs==20.3.0
-    # via
-    #   flake8-pyi
-    #   pytest
```

Comment:
Don't edit this file, it's managed automatically. In this case, `attrs` comes in as a transitive dependency as visible in the comment.